WFLY-9767 Clustering testsuite trimming - redundant testsuite/integra…

rhusar opened the pull request:
…tion/clustering/target/wildfly

Jira
https://issues.jboss.org/browse/WFLY-9767

The testsuite/integration/clustering/target/wildfly is copied by an inherited execution from testsuite/pom.xml, which is then used for templating but remains untouched for the testsuite run. That is not necessary. This saves another ~30 MB.

rhusar committed:
WFLY-9767 Clustering testsuite trimming - redundant testsuite/integration/clustering/target/wildfly

jmesnil commented:
is the failure in the clustering integration test suite related to the PR?

```
Failed to execute goal org.wildfly.plugins:wildfly-maven-plugin:1.2.1.Final:execute-commands (default) on project wildfly-ts-integ-clustering: Invalid JBoss Home directory is not valid: /store/work/tc-work/9ccd5e119c4a65d0/testsuite/integration/clustering/target/wildfly
[05:19:35][Step 2/3] 
```

rhusar commented:
@jmesnil This seems to uncovers a bigger problem, it looks like the https://github.com/wildfly/wildfly/blob/master/testsuite/integration/clustering/pom.xml#L35 are geting run out of order. Maven should run executions for the same phase in order they are defined in pom.xml, but looks like they aren't. This used to be a bug multiple times in the past MNG-2258 or MNG-3719.

rhusar committed:
Workaround maven phases run in arbitrary order as opposed to as defined in a pom

Maven should run executions for the same phase in order they are defined in pom.xml, but looks like they aren't. This used to be a bug multiple times in the past MNG-2258 or MNG-3719.

[13:19:45] :	 [Step 2/3] [INFO] --- maven-antrun-plugin:1.8:run (ts.config-as.clustering) @ wildfly-ts-integ-clustering ---
[13:19:45] :	 [Step 2/3] [INFO] Executing tasks
[13:19:45] :	 [Step 2/3]
[13:19:45] :	 [Step 2/3] main:
[13:19:45] :	 [Step 2/3]
[13:19:45] :	 [Step 2/3] build-clustering:
[13:19:45] :	 [Step 2/3]      [move] Moving 45 files to /store/work/tc-work/9ccd5e119c4a65d0/testsuite/integration/clustering/target/wildfly-1
[13:19:45] :	 [Step 2/3]      [copy] Copying 45 files to /store/work/tc-work/9ccd5e119c4a65d0/testsuite/integration/clustering/target/wildfly-2
[13:19:45] :	 [Step 2/3]      [copy] Copying 45 files to /store/work/tc-work/9ccd5e119c4a65d0/testsuite/integration/clustering/target/wildfly-3
[13:19:45] :	 [Step 2/3]      [copy] Copying 45 files to /store/work/tc-work/9ccd5e119c4a65d0/testsuite/integration/clustering/target/wildfly-4
[13:19:45] :	 [Step 2/3] [INFO] Executed tasks
[13:19:45] :	 [Step 2/3] [INFO]
[13:19:45] :	 [Step 2/3] [INFO] --- wildfly-maven-plugin:1.2.1.Final:execute-commands (default) @ wildfly-ts-integ-clustering ---
[13:19:45]:	 [Step 2/3] Failed to execute goal org.wildfly.plugins:wildfly-maven-plugin:1.2.1.Final:execute-commands (default) on project wildfly-ts-integ-clustering: Invalid JBoss Home directory is not valid: /store/work/tc-work/9ccd5e119c4a65d0/testsuite/integration/clustering/target/wildfly
